javagen/src/main/java/com/azure/autorest/template/ProtocolSampleTemplate.java

```diff
@@ -74,32 +84,39 @@ public void write(ProtocolExample protocolExample, JavaFile javaFile) {
         List<String> clientParameterLines = new ArrayList<>();
         Set<ServiceClientProperty> processedServiceClientProperties = new HashSet<>();
 
+        List<ProxyMethodParameter> proxyMethodParameters = getProxyMethodParameters(method.getProxyMethod(), method.getParameters());
+
         example.getParameters().forEach((parameterName, parameterValue) -> {
```

Comment:
could we change the variable name 'example' to 'proxyMethodExample' to make it less confusion? we have both proxyMethodExample and protocolExample here. The logic here is a bit complex already :)